src/layouts/Dashboard.js

```diff
@@ -361,49 +374,221 @@ export default function Dashboard() {
     tableFilter,
   ]);
 
+  useEffect(() => {
+    if (datastore.comparisonMode && datastore.comparisonMode !== 'Default') {
+      (async () => {
+        setIsLoading(true);
+        // match label to filterOptions key
+        const aliasObj = {
+          payors: 'Payors',
+          healthcareProviders: 'Providers',
+          healthcareCoverages: 'Coverages',
+          healthcarePractitioners: 'Practitioners',
+        };
+        const filterKey = Object.entries(aliasObj)
+          .find(([, label]) => label === datastore.comparisonMode)?.[0];
+        const items = filterKey ? (datastore.filterOptions[filterKey] || []) : [];
+        const allResults = [];
+        for (const item of items) {
+          const search = await filterSearch(
+            false,
+            [item.value],
+            isComposite,
+            datastore.measurementYear,
+          );
+          // map each result to this payor series
+          allResults.push(
+            ...search.dailyMeasureResults.map((r) => ({
+              ...r,
+              measure: item.value,
+            })),
+          );
+        }
+        setCurrentResults(allResults);
+        setDisplayData(allResults);
+        setSelectedMeasures(items.map((i) => i.value));
+        setIsLoading(false);
+      })();
+    }
+  }, [datastore.comparisonMode, datastore.filterOptions]);
```

Comment:
I'm testing this and was running into errors whenever trying to compare things. So I removed this entire useEffect and it fixed all my problems.

Reply:
oh. yeah. that thing. don't worry about that thing.